github/client.go

```diff
@@ -335,27 +335,34 @@ func (client *Client) UploadReleaseAsset(uploadUrl *url.URL, asset *os.File, con
 	return
 }
 
-func (client *Client) CIStatus(project *Project, sha string) (status *octokit.Status, err error) {
-	url, err := octokit.StatusesURL.Expand(octokit.M{"owner": project.Owner, "repo": project.Name, "ref": sha})
+type CIStatusResponse struct {
+	State    string      `json:"state"`
+	Statuses []*CIStatus `json:"statuses"`
```

Comment:
This could be `[]CiStatus`. A array of pointers doesn't seem to buy us anything.